open_event/helpers/data.py

```diff
@@ -1009,6 +1015,9 @@ def edit_event(request, event_id, event, session_types, tracks, social_links, mi
                                                event_id=event.id)
                 db.session.add(social_link)
 
+        for track in tracks:
+            delete_from_db(track, "Tracks Deleted")
```

Comment:
No. When you do this, after edit the tracks are recreated, their IDs change. But the sessions are still having the old non-existent track_ids stored in them. This same issue was there before (#1040) and was fixed by @shivamMg.